### PR TITLE
Acknowledge all Outreachy participants

### PR DESCRIPTION
**Pull request by sgibson91**
We should also acknowledge the community members that made it possible to have Outreachy interns work on the project with us, as this is a lot of invisible work.

### CODE REVIEW

File: docs/_data/outreachy/outreachy_participants.json [hunk 1]
```diff
@@ -17,6 +17,23 @@
...
       }
+    ],
+    "community_members": [
+      {
+        "name": "Min Ragan-Kelley",
+        "github_handle": "minrk",
+        "roles": ["mentor"]
+      },
+      {
+        "name": "Georgiana Dolocon",
```
Comment (GeorgianaElena) [inline docs/_data/outreachy/outreachy_participants.json:28]:
```suggestion
        "name": "Georgiana Dolocan",
```

**Review by GeorgianaElena — APPROVED** [review]
❤️

File: docs/_data/outreachy/outreachy_participants.json (hunk 1 above)
Comment (sgibson91) [reply docs/_data/outreachy/outreachy_participants.json]: So sorry @GeorgianaElena! I think I've fixed this now

**Review by choldgraf — APPROVED** [review]
This looks like a nice improvement to me, thanks for making this work more visible! I think we might want to consider putting these in a dedicated place in the future (e.g. maybe `hub.jupyter.org` can have shout-outs for something like this?) but IMO doesn't need to block on this PR

I feel a little bit awkward noting this, but it would be nice for me to be listed somewhere as well. I didn't participate in Outreachy directly, but I spent quite a lot of time writing grant applications to get funding to cover our first and second rounds with Outreachy. I know that this is a different kind of participation than working directly with interns, but it was a significant amount of work nonetheless and is also somewhat "invisible". It feels like something we should recognize somehow (sorry if this comes across as selfish!). Maybe we should have a different space to give thanks to people that have gotten direct funding for the JupyterHub project? (e.g., if the funding went to NumFocus and not their parent organization).

**Comment by sgibson91** [comment]
No Chris, this is an excellent point!

**Comment by sgibson91** [comment]
Shall we open an issue to brainstorm where to acknowledge and what types of activities?

I also feel we are getting close to the emoji key as well. Is having an emoji key next to someone's name indicating that they write grants (or other activities) "visible enough"?

**Comment by sgibson91** [comment]
@choldgraf I opened https://github.com/jupyterhub/team-compass/issues/624

**Comment by sgibson91** [comment]
I've also added you to the Outreachy docs while we figure out a better system for the team compass https://jupyterhub-outreachy.readthedocs.io/en/latest/contributing/acknowledgements.html

**Pull request merged**